bees/twitterbee/twitterbeefactory.go

```diff
@@ -168,6 +168,11 @@ func (factory *TwitterBeeFactory) Events() []bees.EventDescriptor {
 					Description: "text content of the tweet",
 					Type:        "string",
 				},
+				{
+					Name:        "url",
+					Description: "URL of the tweet",
+					Type:        "string",
```

Comment:
ok, should the name be changed to something like twitt_url to avoid confusion or it's ok as is?